File: template/template.c
#include <stdio.h>

int main(void)
{
    /* TODO edit this code, this code is for https://atcoder.jp/contests/practice/tasks/practice_1 */

    /* param */
    int a;
    int b;
    int c;
    char s[101];
    scanf("%d\n%d %d\n%s", &a, &b, &c, s);

    /* solve */
    int ans = a + b + c;

    /* answer */
    printf("%d %s", ans, s);

    return 0;
}

File: template/template.go
package main

import (
	"fmt"
)

func main() {

	// TODO edit this code, this code is for https://atcoder.jp/contests/practice/tasks/practice_1

	// param
	var a int
	var b, c int
	var s string
	fmt.Scanf("%d\n", &a)
	fmt.Scanf("%d %d\n", &b, &c)
	fmt.Scanf("%s\n", &s)

	// answer
	fmt.Printf("%d %s\n", a+b+c, s)
}
